### PR TITLE
Refactor `_addButton` method of editor

### PR DESCRIPTION
**Pull request by davidculley**
- Condensed and simplified the various if-else-construct.
- Disentangled the logic of a particularly confusing if-else-construct.
- Transformed variables to snake_case (see PEP8) and gave them clearer names.
- Fixed an import warning (`import os`).
- Fixed some f-strings.

### CODE REVIEW

File: qt/aqt/editor.py [hunk 1]
```diff
@@ -287,7 +288,7 @@ def _addButton(
         self,
         icon: str | None,
         cmd: str,
-        tip: str = "",
+        title_attribute: str = "",
```
Comment (dae) [inline qt/aqt/editor.py:291]: This is part of the public API, so changing it could break add-ons.

Comment (davidculley) [reply qt/aqt/editor.py]: Right, I didn't take add-ons into account. I checked only the code base of Anki itself.

Lesson learned, won't happen again. 🙂

**Comment by dae** [comment]
Thanks!

**Comment by davidculley** [comment]
I'm glad I could help!

**Pull request merged**